assets/js/blocks/checkout/edit.tsx

```diff
@@ -109,6 +111,7 @@ const BlockSettings = ( {
 				/>
 			</PanelBody>
 			<CartCheckoutFeedbackPrompt />
+			<CheckoutSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the Name & description panel before the Block's settings panels (i.e., Let's move it to line 69).